spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnGraph.scala

```diff
@@ -55,7 +55,7 @@ class DnnGraph(
       val out = output.toTensor[Float]
       // for grey image, input should be 3 dims and the first dim should be batch size
       // for non grey image, input should be 4 dims and the first dim should be batch size
-      require(in.nDimension() == 4 || in.nDimension() == 3,
+      require(in.nDimension() == 4 || in.nDimension() == 3 || in.nDimension() == 2,
```

Comment:
Please add comment about the condition of `in.nDimension() == 2`

Reply:
ok, changed